server/log_operation_manager.go

```diff
@@ -312,30 +312,31 @@ func (l *LogOperationManager) getLogsAndExecutePass(ctx context.Context) error {
 
 				start := time.Now()
 				count, err := l.logOperation.ExecutePass(ctx, logID, &l.info)
+				if err != nil {
+					glog.Warningf("ExecutePass(%v) failed: %v", logID, err)
+					return
```

Comment:
Should this be `break` rather than `return` ?  The worker goroutine could carry on round the loop and process another logID.